packages/teams-js/test/public/video.spec.ts

```diff
@@ -650,4 +651,92 @@ describe('video', () => {
       expect(messageForRegister?.args).toEqual([false, effectId, 'InvalidEffectId']);
     });
   });
-});
+
+  describe('video.mediaStream', () => {
+    it.each([() => framedPlatformMock.uninitializeRuntimeConfig(), () => framelessPlatformMock.uninitializeRuntimeConfig()])
+    ('should throw if called before initialization', (uninitializeRuntime) => {
+      uninitializeRuntime();
+      expect(() => video.mediaStream.isSupported()).toThrowError(new Error(errorLibraryNotInitialized));
+    });
+
+    it.each([() => framedPlatformMock.initializeWithContext(FrameContexts.content), () => framelessPlatformMock.initializeWithContext(FrameContexts.content)])
+    ('should return false if texture stream is not supported', async (initializeWithContext) => {
```

Comment:
I don't see any tests that are testing what happens when the runtime object doesn't include mediaStream (see calls to setRuntimeConfig elsewhere in this file). Do you have tests that are checking both that as well as whether or not textureStream is supported?

Reply:
(I don't think "should throw if called before initialization" is quite the same thing)